web/cobrands/hackney/assets.js

```diff
@@ -0,0 +1,275 @@
+  http_options: {
+      params: {
+          TYPENAME: "parking:pay_display_machine_liberator",
+      }
+  },
+  asset_category: "Pay & Display Machines",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "recycling:estate_recycling_bin",
+      }
+  },
+  asset_category: "Bin Contamination",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "amenity:sport_facility",
+      }
+  },
+  asset_category: "Sport Area",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "greenspaces:park_bench",
+      }
+  },
+  asset_category: "Park Furniture (bench)",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "greenspaces:park_bin",
+      }
+  },
+  asset_category: "Park Furniture (bin)",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "greenspaces:park_picnic_bench",
+      }
+  },
+  asset_category: "Park Furniture (picnic bench)",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "amenity:public_toilet",
+      }
+  },
+  asset_category: "Toilets",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "transport:bike_hangar_scheme",
+      }
+  },
+  asset_category: "Cycle Hangars",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "greenspaces:park_bench",
+      }
+  },
+  asset_category: "Benches",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "greenspaces:park_bench",
+      }
+  },
+  asset_category: "Benches",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "amenity:public_toilet",
+      }
+  },
+  asset_category: "Toilets",
+  attributes: {}
+});
+
+/** These layers are served directly from Alloy: */
+
+// View all layers with something like:
+// curl https://tilma.staging.mysociety.org/resource-proxy/proxy.php\?https://hackney.assets/ | jq '.results[] | .layer.code, ( .layer.styles[] | { id, name } ) '
+var layers = [
+  {
+    "categories": ["Street Lighting", "Lamposts"],
+    "item_name": "street light",
+    "layer_name": "Street Lights",
+    "styleid": "5d308d57fe2ad8046c67cdb5",
+    "layerid": "layers_streetLightingAssets"
+  },
+  {
+    "categories": ["Illuminated Bollards", "Non-illuminated Bollards"],
+    "item_name": "bollard",
+    "layer_name": "Bollards",
+    "styleid": "5d308d57fe2ad8046c67cdb9",
+    "layerid": "layers_streetLightingAssets"
+  },
+  {
+    "categories": ["Benches"],
```

Comment:
Just to check, does the displaying of multiple asset layers for the same category work okay selection-wise?

Reply:
Yep, it appears to! Right now we don't do anything with the feature attributes for the WFS layers, but having tested it locally it works as expected.